### DISCUSSION TITLE
Weird display on table list "show per page"

### DISCUSSION THREAD

**Discussion by jinbatsu** [filamentphp/filament#9019] [Help] [answered]
In the Pagination display based on width screen responsive.
But in my current screen the display is not so efficients.

![2023-10-11_11-46-38](https://github.com/filamentphp/filament/assets/103924/6236f79c-24fe-4043-b322-9ba82da3b94a)

Is there any configuartion or manipulate this pagination display?

**Comment by sinnbeck** [comment, marked as answer]
Add a custom theme that extends the default and add this to the theme.css file and run `npm run build`

```css
.fi-pagination {
    @apply lg:flex lg:justify-between;
}

**Reply by jinbatsu** [reply]
After updated to the latest, suddently the default only display (**prev** and **next**), which is great for me.
Here already discussed: https://github.com/filamentphp/filament/pull/9162

![2023-10-26_00-53-13](https://github.com/filamentphp/filament/assets/103924/42599836-a913-46fa-94a8-9b740cbfcf14)
